multi-node/aws/cluster.yaml.example

```diff
@@ -39,3 +39,24 @@ externalDNSName:
 # of the kube-aws tool. This parameter is typically
 # overwritten only for development purposes.
 #artifactURL: https://coreos-kubernetes.s3.amazonaws.com/<VERSION>
+
+# CIDR for kubernetes vpc
+# vpcCidr: "10.0.0.0/16"
+
+# CIDR for kubernetes subnet
+# instanceCidr: "10.0.0.0/24"
+
+# IP Address for controller in kubernetes subnet
+# controllerIp: 10.0.0.50
+
+# CIDR for all service ip addresses
+# serviceCidr: "10.3.0.0/24"
+
+# CIDR for all pod ip addresses
+# podCidr: "10.2.0.0/16"
+
+# ip address of k8s controller service (must be contained by serviceCidr)
```

Comment:
One more nit - please don't use `k8s` anywhere in code/config/docs. It's not a well-understood abbreviation of Kubernetes.